src/app/(proper_react)/redesign/(authenticated)/user/dashboard/fix/ResolutionContentLayout.tsx

```diff
@@ -0,0 +1,77 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+"use client";
+
+import styles from "./ResolutionContentLayout.module.scss";
+import Image from "next/image";
+import { Button } from "../../../../../../components/server/Button";
+import Link from "next/link";
+import { useL10n } from "../../../../../../hooks/l10n";
+import { ClockIcon } from "../../../../../../components/server/Icons";
+import { ReactNode } from "react";
+
+type ResolutionContentLayoutProps = {
+  type: "highRisk" | "leakedPasswords" | "securityRecommendations";
+  title: string;
+  illustration: {
+    img: string;
+    alt: string;
+  };
+  cta: {
+    label: string;
+    onClick: () => void;
+    skip?: string;
+  };
+  estimatedTime?: number;
+  children: ReactNode;
+};
+
+// TODO: Add test once routes from MNTOR-1700 is available
+/* c8 ignore start */
+export const ResolutionContentLayout = (
+  props: ResolutionContentLayoutProps
+) => {
+  const l10n = useL10n();
+
+  return (
+    <div className={styles.container}>
```

Comment:
Looking at the designs I think we might also want the toolbar here [like on the dashboard](https://github.com/mozilla/blurts-server/blob/68a23a1c358092ed5b635c9d331379b977fae221/src/app/(proper_react)/redesign/(authenticated)/user/dashboard/View.tsx#L227-L234) - though I'm not sure whether including the `<TabList>` there makes sense. But we should probably at least give the user the opportunity to log out :P

Reply:
Unless Zachery disagrees of course - I suppose they can also hit the `⨯` first.